Formula/ktor.rb

```diff
@@ -0,0 +1,18 @@
+class Ktor < Formula
+  desc "CLI tool that creates porjects with ktor framework setup"
+  homepage "https://github.com/ktorio/ktor-cli"
+  url "https://files.pkg.jetbrains.space/public/p/ktor-cli-tool/mac-release/ktor-0.0.1.tar.gz"
```

Comment:
This should probably be a sourcce build from the repo rather than a macOS specific download that won't work on Linux.

Reply:
For linux we are considering to publish the tool to apt-get and other package managers, but if it's possible to also support Homebrew it would be awesome as well!

Is it possible to maybe specify different URLs for different OS?

Reply:
> Is it possible to maybe specify different URLs for different OS?

No, it should build from source instead.

Reply:
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/kotlin-language-server.rb and others seem to manage just fine.

Reply:
As I can see from their GitHub (https://github.com/fwcd/kotlin-language-server) they are not using Kotlin/Native. Instead, they use Kotlin/Jvm which is absolutely easy to build.

Moreover, in Kotlin/Native there are different Gradle commands to build for MacOs, Linux, Windows and/or other target platform:
For MacOs: `linkReleaseExecutableMacosX64`
For Linux: `linkReleaseExecutableLinuxX64` 

Our idea is to create the tool using Kotlin/Native, maintain its binary releases on our TeamCity automation and make them public for MacOs via Homebrew, for Linux via Apt and Pacman, for Windows via WinGet

Please tell your suggestions if you have any, it will be appreciated

Reply:
> As I can see from their GitHub (https://github.com/fwcd/kotlin-language-server) they are not using Kotlin/Native. Instead, they use Kotlin/Jvm which is absolutely easy to build.

But it is build with gradle tasks, which seemed to be the problem in your earlier comment.

> Moreover, in Kotlin/Native there are different Gradle commands to build for MacOs, Linux, Windows and/or other target platform

So add an `if` statement? 

> Our idea is to create the tool using Kotlin/Native, maintain its binary releases on our TeamCity (https://ktor.teamcity.com/project/Ktor_ProjectKtorCLI?mode=builds) automation and make them public for MacOs via Homebrew, for Linux via Apt and Pacman, for Windows via WinGet

That does not meet the requirement in https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae.